packages/backend/package.json

```diff
@@ -17,8 +17,10 @@
 		"jest": "cross-env NODE_ENV=test node --experimental-vm-modules --experimental-import-meta-resolve node_modules/jest/bin/jest.js --forceExit --runInBand",
 		"jest-and-coverage": "cross-env NODE_ENV=test node --experimental-vm-modules --experimental-import-meta-resolve node_modules/jest/bin/jest.js --coverage --forceExit --runInBand",
 		"jest-clear": "cross-env NODE_ENV=test node --experimental-vm-modules --experimental-import-meta-resolve node_modules/jest/bin/jest.js --clearCache",
-		"test": "pnpm jest",
-		"test-and-coverage": "pnpm jest-and-coverage"
+		"vitest": "cross-env NODE_ENV=test vitest --no-threads --run",
+		"coverage": "cross-env NODE_ENV=test vitest run --coverage --no-threads",
+		"test": "pnpm vitest",
+		"test-and-coverage": "pnpm coverage"
```

Comment:
Replaced Jest with Vitest for experimental purposes. As far as I know, Vitest uses esbuild, and there may be some issues when it comes to decorator metadata, which is used by TypeORM. Watch mode is disabled for now ([`--run`](https://vitest.dev/guide/cli.html#options)), otherwise `Something went wrong installing the "sharp" module` will be thrown for some reason.